l10n_it_partially_deductible_vat/__openerp__.py

```diff
@@ -24,27 +24,25 @@
 ##############################################################################
 
 {
-    "name" : "Italy - Partially Deductible VAT",
-    "version" : "0.1",
-    "depends" : ['account'],
-    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
+    "name": "Italy - Partially Deductible VAT",
+    "version": "0.1",
+    "depends": ['account'],
+    "author": "OpenERP Italian Community,Odoo Community Association (OCA),"
+              "SimplERP srl",
     "description": """
```

Comment:
@dcorio since https://github.com/odoo/odoo/commit/ec0b770ed3c4f4fbdf0f24930aaa05bcc5cb0b53 the description key could be extracted in README.rst.